src/mixins/filing-template-mixin.ts

```diff
@@ -375,7 +375,11 @@ export default class FilingTemplateMixin extends Mixins(AmalgamationMixin, DateM
           affidavitFileKey: this.getExistingBusinessInfo?.affidavitFileKey,
           affidavitFileName: this.getExistingBusinessInfo?.affidavitFileName
         },
-        authorization: this.getContinuationAuthorization,
+        authorization: {
+          files: this.getContinuationAuthorization.files,
+          date: this.getContinuationAuthorization.date,
+          expiryDate: this.getContinuationAuthorization.expiryDate || undefined // can't be null
```

Comment:
When this JS object is converted to JSON, `undefined` properties are not included.